src/aiodynamo/models.py

```diff
@@ -129,13 +129,13 @@ def encode(self) -> EncodedLocalSecondaryIndex:
 
 @dataclass(frozen=True)
 class GlobalSecondaryIndex(LocalSecondaryIndex):
-    throughput: Throughput
+    throughput: Optional[Throughput]
 
     def encode(self) -> EncodedGlobalSecondaryIndex:
         # Cast due to https://github.com/python/mypy/issues/4122
         return cast(
             EncodedGlobalSecondaryIndex,
```

Comment:
please also update the type definition of `EncodedGlobalSecondaryIndex`, it should now be [`total=False`](https://docs.python.org/3/library/typing.html#typing.TypedDict) since `ProvisionedThroughput` is not always set.

Reply:
done, thanks!